packages/manager/apps/dedicated/client/app/billing/autoRenew/cancel-resiliation/cancel-resiliation.routing.js

```diff
@@ -0,0 +1,62 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.account.billing.autorenew.cancel-resiliation', {
+    url: '/cancel-resiliation?serviceId&serviceType',
+    views: {
+      modal: {
+        component: 'billingCancelResiliation',
+      },
+    },
+    layout: 'modal',
+    translations: { value: ['.'], format: 'json' },
+    resolve: {
+      goBack: /* @ngInject */ (goToAutorenew) => goToAutorenew,
+      cancelResiliation: /* @ngInject */ (
+        BillingAutoRenew,
+        engagement,
+        hasEndRuleStrategies,
+        setReactivateEngagementStrategy,
+      ) => (service) => {
+        if (engagement && hasEndRuleStrategies) {
+          return setReactivateEngagementStrategy();
+        }
+
+        service.cancelResiliation();
+        return BillingAutoRenew.updateService(service);
+      },
+      engagement: /* @ngInject */ (Server, service) =>
+        (service.canHaveEngagement()
+          ? Server.getSelected(service.serviceId)
```

Comment:
I wonder if this condition has changed and other services can have an engagement